### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by mikeshaw-stripe**
Updating the README to include steps on how to onboard to your Stripe account

- Removing the existing transactions and Stripe account from the DB
- Adding the 10% marketplace fee into the payment so that you can see it on your Stripe Account

### CODE REVIEW

File: db/database.json [hunk 1]
```diff
@@ -14,10 +14,7 @@
       "lastName": "Owner",
       "email": "owner@kavholm.com",
       "password": "$2b$10$fDU/gKP8pL5LeozB3WBodu/zJodwplgqqAk2xkMYavFGHCcJ0/IEi",
-      "avatar": "//images.unsplash.com/photo-1506863530036-1efeddceb993?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2741&q=80",
-      "stripe": {
-        "stripeUserId": "acct_1G1Os2APkNRJ6fly"
```
Comment (auchenberg-stripe) [inline db/database.json]: We need to keep this for the demo on kavholm.com to work

**Comment by auchenberg-stripe** [comment]
LGTM, with the exception of removing the `stripeUserId` in the database.json

File: db/database.json (hunk 1 above)
Comment (mikeshaw-stripe) [reply db/database.json]: Ah! Will re-add :)

**Comment by auchenberg-stripe** [comment]
LGTM

**Pull request merged**